rust/processor/src/db/common/models/token_v2_models/raw_token_claims.rs

```diff
@@ -0,0 +1,215 @@
+// Copyright © Aptos Foundation
+// SPDX-License-Identifier: Apache-2.0
+
+// This is required because a diesel macro makes clippy sad
+#![allow(clippy::extra_unused_lifetimes)]
+#![allow(clippy::unused_unit)]
+
+use crate::{
+    db::postgres::models::{
+        token_models::{token_utils::TokenWriteSet, tokens::TableHandleToOwner},
+        token_v2_models::v2_token_activities::TokenActivityHelperV1,
+    },
+    utils::util::standardize_address,
+};
+use ahash::AHashMap;
+use aptos_protos::transaction::v1::{DeleteTableItem, WriteTableItem};
+use bigdecimal::{BigDecimal, Zero};
+use serde::{Deserialize, Serialize};
+
+// Map to keep track of the metadata of token offers that were claimed. The key is the token data id of the offer.
+// Potentially it'd also be useful to keep track of offers that were canceled.
+pub type TokenV1Claimed = AHashMap<String, TokenActivityHelperV1>;
+
+#[derive(Clone, Debug, Deserialize, Eq, PartialEq, Serialize)]
+pub struct RawCurrentTokenPendingClaim {
+    pub token_data_id_hash: String,
+    pub property_version: BigDecimal,
+    pub from_address: String,
+    pub to_address: String,
+    pub collection_data_id_hash: String,
+    pub creator_address: String,
+    pub collection_name: String,
+    pub name: String,
+    pub amount: BigDecimal,
+    pub table_handle: String,
+    pub last_transaction_version: i64,
+    pub last_transaction_timestamp: chrono::NaiveDateTime,
+    pub token_data_id: String,
+    pub collection_id: String,
+}
+
+impl Ord for RawCurrentTokenPendingClaim {
```

Comment:
I thought only the parquet models require `Ord` and `PartialOrd`?

Reply:
sort is done for raw models in the parsing logic